src/main/java/rs/teslaris/core/repository/RefreshTokenRepository.java

```diff
@@ -0,0 +1,12 @@
+package rs.teslaris.core.repository;
+
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import rs.teslaris.core.model.RefreshToken;
+
+public interface RefreshTokenRepository extends JpaRepository<RefreshToken, Integer> {
```

Comment:
it is not mandatory, but we should be consistent. I have noticed that you added @Repository annotation in other JpaRepositories

Reply:
You are right, I don't know how this even worked in the first place without @Component or @Repository annotation... I will add it right away.